h2o/h2o.sh

```diff
@@ -54,6 +54,11 @@ EOF
 }
 
 function main() {
+  if [[ "${DATAPROC_VERSION}" == "1.5" ]]; then
+    echo "Dataproc 1.5 not supported. Please use Dataproc 1.3, 1.4 or 2.0+"
```

Comment:
I think we need to add similar condition to skip test for 1.5 images, if we have it?

Reply:
Fixed